### PR TITLE
Expand lzip and zip files correctly

### PR DESCRIPTION
**Pull request by JL2210**
Fix #3231

### CODE REVIEW

File: crew [hunk 1]
```diff
@@ -449,8 +449,8 @@ def unpack (meta)
     when /\.zip$/i
       puts "Unpacking archive using 'unzip', this may take a while..."
       _verbopt = @opt_verbose ? 'v' : 'qq'
-      system "unzip", _verbopt, "-d", "#{extract_dir}", meta[:filename]
-    when /\.(tar(\.(gz|bz2|xz))?|tgz|tbz)$/i
+      system "unzip", '-' + _verbopt, "-d", "#{extract_dir}", meta[:filename]
```
Comment (uberhacker) [inline crew]: Why not just include the dash in `_verbopt`?

Comment (uberhacker) [reply crew]: We also need a version bump in `lib/const.rb`.

Comment (JL2210) [reply crew]: The dash in `_veropt` would be slightly redundant. Version bumped.

Comment (JL2210) [reply crew]: That seems fair. Give me a minute...

**Pull request merged**